src/sentry/api/endpoints/organization_events.py

```diff
@@ -281,8 +281,15 @@ def get(self, request: Request, organization) -> Response:
 
         referrer = request.GET.get("referrer")
         use_metrics = (
-            features.has(
-                "organizations:mep-rollout-flag", organization=organization, actor=request.user
+            (
+                features.has(
+                    "organizations:mep-rollout-flag", organization=organization, actor=request.user
+                )
+                and features.has(
+                    "organizations:server-side-sampling",
+                    organization=organization,
+                    actor=request.user,
```

Comment:
Thinking about it, I'm not sure we're actually going to remove the chain on `organizations:server-side-sampling` so doing the bool is probably redundant, just more cleanup later? Thoughts?

Reply:
(if i understand your comment) The chain only happens on org serialization, so it'll be necessary here as long as we're using the mep-rollout-flag

Reply:
Oh true, 👍